src/routes/blog/1/+page.svelte

```diff
@@ -17,26 +22,44 @@
 </script>
 
 <Blog_Post {post}>
-	<p>
-		This blog was created with <code>fuz_blog</code>, a SvelteKit library for making blogs for
-		developers. (<a href="https://blog.fuz.dev">blog.fuz.dev</a>,
-		<a href="https://github.com/ryanatkn/fuz_blog">source code</a>)
-	</p>
-	<p>
-		If you're interested in the full videos where I made the software, extracting it from my
-		personal blog:
-	</p>
-	<ol>
-		<li>
-			<a href="https://www.youtube.com/watch?v=wP9qghWkTLc"
-				>Coding a blog creation library for SvelteKit developers from scratch</a
-			>
-		</li>
-		<li>
-			<a href="https://www.youtube.com/watch?v=dh_6Vyjbuiw"
-				>Extracting fuz_blog from my personal blog and publishing to npm</a
-			>
-		</li>
-	</ol>
-	<p>Stay tuned for more.</p>
+	<section>
+		<p>
+			This blog was created with <code>fuz_blog</code>, a SvelteKit library for making blogs for
+			developers. (<a href="https://blog.fuz.dev">blog.fuz.dev</a>,
+			<a href="https://github.com/ryanatkn/fuz_blog">source code</a>)
+		</p>
+		<p>
+			If you're interested in the full videos where I made the software, extracting it from my
+			personal blog:
+		</p>
+		<ol>
+			<li>
+				<a href="https://www.youtube.com/watch?v=wP9qghWkTLc"
+					>Coding a blog creation library for SvelteKit developers from scratch</a
+				>
+			</li>
+			<li>
+				<a href="https://www.youtube.com/watch?v=dh_6Vyjbuiw"
+					>Extracting fuz_blog from my personal blog and publishing to npm</a
+				>
+			</li>
+		</ol>
+		<p>Stay tuned for more.</p>
+	</section>
+	<section>
+		<h2>Comments</h2>
+		<!-- TODO post.comments.url instead -->
+		<!-- TODO the storage key is weird -->
+		<!-- TODO use local cache in dev -->
+		<!-- TODO remove `:any` when fuz_mastodon types are fixed -->
+		<Toot
+			url={post.comments.url}
+			replies
+			storage_key="1_comments"
+			initial_autoload={true}
```

Comment:
this was broken, fixed here: https://github.com/ryanatkn/webdevladder.net/commit/ae9918006dbb970329c1eda8f6a655035e6cbee3